service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2234,6 +2264,14 @@ components:
           items:
             $ref: "#/components/schemas/CriteriaGroupSection"
 
+    CohortCloneInfo:
```

Comment:
It occurred to me now that we need an optional `destinationStudyId` in these clone endpoints also (data feature set & cohort). Cloning within a study is nice to have, but the immediate use case is calling these endpoints as part of cloning an entire study, so we'd need the destination study ids. Sorry I didn't catch that in the data feature set PR!

Reply:
Adding it in a separate PR.

Reply:
https://github.com/DataBiosphere/tanagra/pull/992